.circleci/conditional_skip.sh

```diff
@@ -6,12 +6,14 @@ set +e
 
 SKIP_TAG="\[skip tests\]"
 
-if [[ ! -z ${CIRCLE_TAG} ]]; then
+# shellcheck disable=SC2154
+if [[ -n ${CIRCLE_TAG} ]]; then
```

Comment:
If you want to avoid these disables, you can use `${CIRCLE_TAG:?}` to get an error at runtime if it is unset. The shellcheck should not complain, anymore.

Reply:
Great, didn't know about that one.
It even supports a custom message. Cool!

Reply:
In this case we can't use it though since `CIRCLE_TAG` not being set is an expected case.
Where it made sense I did update the other scripts though.